mathparse/mathparse.py

```diff
@@ -96,7 +96,18 @@ def replace_word_tokens(string, language):
 
     # Replace operator words with numeric operators
     operators = words['binary_operators'].copy()
-    operators.update(words['unary_operators'])
+    if u'unary_operators' in words:
+        operators.update(words['unary_operators'])
+
+    # Ensures unicode string processing
```

Comment:
I'm not sure if decoding the string is necessary. Was there a case where this was failing?